src/open_company_web/components/topic.cljs

```diff
@@ -2,15 +2,154 @@
+          actual (last sorted-data)
+          currency (:currency options)
+          cur-symbol (utils/get-symbol-for-currency-code currency)
+          cash-val (str cur-symbol (utils/format-value (:cash actual)))
+          actual-label (str "as of " (finances-utils/get-as-of-string (:period actual)))]
+      (dom/div {:class (utils/class-set {:topic-headline true
+                                         :topic-headline-finances true
+                                         :group true
+                                         :collapse (:expanded data)})}
+        (dom/div {:class "target-actual-container"}
+          (dom/div {:class "actual-container"}
+            (dom/h3 {:class "actual green"} cash-val)
+            (dom/h3 {:class "actual-label gray"} actual-label)))))))
+
+(defcomponent topic-headline-growth [data owner options]
+  (render [_]
+    (let [metric-info (first (:metrics data))
+          metric-data (filter #(= (:slug metric-info) (:slug %)) (:data data))
+          sort-pred (utils/sort-by-key-pred :period true)
+          sorted-metric (vec (sort sort-pred metric-data))
+          interval (:interval metric-info)
+          metric-unit (:unit metric-info)
+          fixed-cur-unit (when (= metric-unit "currency")
+                           (utils/get-symbol-for-currency-code (:currency options)))
+          unit (when (= metric-unit "%") "%")]
+      (dom/div {:class (utils/class-set {:topic-headline true
+                                         :topic-headline-growth true
+                                         :group true
+                                         :collapse (:expanded data)})}
+        (dom/div {:class "chart-header-container"}
+          (dom/div {:class "target-actual-container"}
+            (dom/div {:class "actual-container"}
+              (dom/h3 {:class "actual blue"} (:name metric-info)))))
+        (om/build column-chart
+                  (growth-utils/get-chart-data sorted-metric
+                                               fixed-cur-unit
+                                               (:slug metric-info)
+                                               (:name metric-info)
+                                               unit
+                                               interval)
+                  {:opts {:chart-height 50 :chart-width 300 :chart-navigation false}})))))
 
 (defcomponent topic [data owner options]
 
+  (init-state [_]
+    {:expanded false})
+
   (render [_]
     (let [company-data (:company-data data)
           section (keyword (:section-name options))
-          section-data (company-data section)]
-      (dom/div {:class "col-xs-12 topic"}
+          section-data (company-data section)
+          expanded (om/get-state owner :expanded)
+          section-body (get-body section-data section)]
+      (dom/div #js {:className "topic"
+                    :ref "topic"
+                    :onClick #(let [$topic (.$ js/window (om/get-ref owner "topic"))
```

Comment:
This is complicated enough it's better as a named function rather than an inline anonymous function.